eng/pipelines/release-cli.yml

```diff
@@ -738,13 +738,15 @@ stages:
               OSVmImage: MMS2022
               TestShell: pwsh
               TestInstallCommand: >
+                $ErrorActionPreference = 'Stop';
```

Comment:
Does `$ErrorActionPreference` need to be specified if the `pwsh` step uses `stop` as the default?

Reply:
I'm not sure, but it didn't catch the error here: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2710732&view=logs&j=010e8c4c-af76-591c-49f9-92bd8fc0534d&t=8dbeac95-0b19-532b-6bc4-b45b90c1a558

Reply:
I do think this works, let's revisit this later if necessary.